conan/api/subapi/list.py

```diff
@@ -100,6 +100,7 @@ def select(self, pattern, package_query=None, remote=None):
         if "*" in pattern.ref or not pattern.version or \
                 (pattern.package_id is None and pattern.rrev is None):
             refs = self.conan_api.search.recipes(pattern.ref, remote=remote)
+            refs = sorted(refs)  # Order alphabetical and older versions first
```

Comment:
can above function accept sorting, so list returned already in the right order from the very beginning? same for line 122